apps/io-func-wallet-solution-backend/src/entity-configuration.ts

```diff
@@ -0,0 +1,114 @@
+import * as t from "io-ts";
+import * as TE from "fp-ts/TaskEither";
+import * as E from "fp-ts/Either";
+import * as RTE from "fp-ts/ReaderTaskEither";
+import { pipe } from "fp-ts/function";
+import { sequenceS } from "fp-ts/lib/Apply";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { UrlFromString } from "@pagopa/ts-commons/lib/url";
+
+import { validate } from "./validation";
+import {
+  GRANT_TYPE_KEY_ATTESTATION,
+  LoA,
+  RELATIVE_TOKEN_ENDPOINT,
+  TOKEN_ENDPOINT_AUTH_METHOD_SUPPORTED,
+  getLoAUri,
+} from "./wallet-provider";
+import { JwkPublicKey } from "./jwk";
+import { Signer } from "./signer";
+import { EntityConfigurationToJwtModel } from "./encoders/entity-configuration";
+
+export const FederationEntityMetadata = t.type({
+  basePath: UrlFromString,
+  organizationName: NonEmptyString,
+  homePageUri: UrlFromString,
+  policyUri: UrlFromString,
+  tosUri: UrlFromString,
+  logoUri: UrlFromString,
+});
```

Comment:
**suggestion(documentation)**: Consider add a comment to each parameter so that it's documented why they are needed.

Reply:
> This is OIDC Federation standard metadata

That alone could be a useful comment

Reply:
> > This is OIDC Federation standard metadata

That alone could be a useful comment. Don't take for granted readers have the same knowledge as you 😉